web/templates/settings/uploaded_files_list.hbs

```diff
@@ -20,17 +20,18 @@
     <td class="upload-size" >{{ size_str }}</td>
     <td class="actions">
         <span class="edit-attachment-buttons">
-            <a type="submit" href="/user_uploads/{{path_id}}" class="btn no-style" title="{{t 'Download file' }}" id="download_attachment" download>
-                <i class="fa fa-download sea-green" aria-hidden="true"></i>
+            <a type="submit" href="/user_uploads/{{path_id}}" class="btn no-style" id="download_attachment" download>
+                <i class="fa fa-download sea-green tippy-zulip-delayed-tooltip" aria-hidden="true" data-tippy-content="{{t 'Download file' }}"></i>
```

Comment:
Why is the tooltip moved from the outer `a` element ot the inner `i` element here? I don't see a strong reason to prevent one or the other.